core/api_requests.py

```diff
@@ -1,7 +1,8 @@
 import requests
 from core.queries import (query_capsules, query_capsule, query_company,
-                          query_histories, query_history, query_missions,
```

Comment:
## pt

Não remover  as missoes ou o comando de missoes nao funcionará

## en

dont remove the missions_query, otherwise de missions command will not work properly